plutus-metatheory/src/Scoped.lagda

```diff
@@ -44,48 +44,26 @@ data ScopedTy n where
 Tel⋆ : ℕ → ℕ → Set
 Tel⋆ n m = Vec (ScopedTy n) m
 
--- contexts
+-- contexts for type and term variables
 
 data Weirdℕ : ℕ → Set where
   Z : Weirdℕ 0
-  S : ∀{n} → Weirdℕ n  → Weirdℕ n
-  T : ∀{n} → Weirdℕ n → Weirdℕ (suc n)
+  S : ∀{n} → Weirdℕ n  → Weirdℕ n -- term variable
+  T : ∀{n} → Weirdℕ n → Weirdℕ (suc n) -- type variable
 
--- variables
+-- variable counter for both types and terms.
 
 data WeirdFin : ∀{n} → Weirdℕ n → Set where
   Z : ∀{n}{w : Weirdℕ n} → WeirdFin (S w)
+  -- add a term variable
   S : ∀{n}{w : Weirdℕ n} → WeirdFin w → WeirdFin (S w)
+  -- add a type variable
   T : ∀{n}{w : Weirdℕ n} → WeirdFin w → WeirdFin (T w)
 
-
--- what is this for?
--- there are two meaningful things here:
--- 1. one to literally convert the number
--- 2. to extract a type context from a term one
--- this looks like (1)
-
-wtoℕ : ∀{n} → Weirdℕ n → ℕ
```

Comment:
these seem to be dead code but I'm not sure if you are using these to explain something in the paper @jmchapman ? Let me know if I should keep them in.

Reply:
on second thought I've dropped that commit in this PR because those aren't really changes that are relevant to this ticket afterall.